.github/workflows/up_ex_doc_version.yml

```diff
@@ -0,0 +1,49 @@
+---
+name: up_ex_doc_version
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Run daily at midnight
+  workflow_dispatch:
+  
+
+jobs:
+  test:
+    name: Update ex_doc version
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Get current version
+        run: |
+            CURR=`curl -s https://hex.pm/api/packages/ex_doc | jq -r '.latest_stable_version' \
+                  jq -r ".[0].name"
+            TITLE="\`ex_doc\` ${CURR} released"
+            PR_EXISTS=`curl -s https://api.github.com/repos/starbelly/rebar3_ex_doc/pulls?state=all%26per_page=100 | grep -c "\"title\": \"${TITLE}\""`
```

Comment:
```suggestion
            PR=$(curl -s https://api.github.com/repos/${GITHUB_REPOSITORY}/pulls?state=all%26per_page=100 | jq ".[] | select(.title==\"${TITLE}\")")
```

Reply:
I ease running in forks (by means of `$GITHUB_REPOSITORY`) and also use `jq` instead of `grep`.